privacyidea/lib/tokens/hotptoken.py

```diff
@@ -298,6 +298,11 @@ def get_init_detail(self, params=None, user=None):
                                                   "value": oath_url,
                                                   "img": create_img(oath_url)
                                                   }
+                except KeyError as ex:
+                    audit_entry = self.find_most_recent_audit_entry(action="POST /auth")
+                    log.error("{0!s}".format((traceback.format_exc())))
+                    log.error('Unknown Tag {0!r} in one of your policy definition: {1!r}'
```

Comment:
You can remove {1|r}, since we can not know the name of the policy at this point.
I think it is fine, so simply have the wrong tag!